tests/scale/test_load.py

```diff
@@ -76,7 +81,10 @@ def result(self) -> bool:
 
     def run(self) -> None:
         try:
+            start = time.time()
             super().run()
+            end = time.time()
+            TIMINGS["deployments"][self.name] = end - start;
```

Comment:
This should use the `lock`:

```
with lock:
    TIMINGS[...] = end - start
```

Reply:
since each key is independent they won't get to a point where they overwrite each others data